frontend/src/components/widgets/Multiselect/Multiselect.tsx

```diff
@@ -132,21 +140,28 @@ class Multiselect extends React.PureComponent<Props, State> {
     })
   }
 
-  private generateNewState(data: OnChangeParams): State {
+  private generateNewState(
+    data: OnChangeParams,
+    overMaxSelections: boolean | undefined
```

Comment:
Is this parameter now unnecessary?

Reply:
Great catch. I need to relook the code after making changes like this. I'll make sure to do that next time.